web/src/components/netflow-overview/netflow-overview.tsx

```diff
@@ -101,7 +101,12 @@ export const NetflowOverview: React.FC<NetflowOverviewProps> = ({
 
   if (error) {
     return <LokiError title={t('Unable to get overview')} error={error} />;
-  } else if (_.isEmpty(metrics) || !totalMetric) {
+  } else if (
+    _.isEmpty(metrics) &&
+    _.isEmpty(droppedMetrics) &&
+    _.isEmpty(dnsLatencyMetrics) &&
+    _.isEmpty(dnsRCodeMetrics)
```

Comment:
do we need to also add `|| !totalMetric` to this condition ?

Reply:
I made `totalMetrics` optionnal here and manage these by graph as it depends of the kebab options of each graph.

Reply:
`kebab options` ?!? :)

Reply:
Yeah true story ! And it's not something you eat :rofl: 
https://github.com/netobserv/network-observability-console-plugin/blob/main/web/src/components/netflow-overview/panel-kebab.tsx#L208

Mainly in these you can select to [compare your rates against total / total drops for exmple](https://github.com/netobserv/network-observability-console-plugin/blob/main/web/src/components/netflow-overview/netflow-overview.tsx#L416)

Reply:
I will review `kebab` pieces at lunch time :)
/LGTM